packages/client-core/src/user/components/Dashboard/DashboardItems.tsx

```diff
@@ -23,6 +24,11 @@ export const SidebarItems = (allowedRoutes) => [
     path: '/admin',
     icon: <DashboardIcon style={{ color: 'white' }} />
   },
+  allowedRoutes.admin && {
```

Comment:
Can you make new scopes `server:read` and `server:write`, and have those control access to the server status? I'd prefer that the `admin:admin` scope just indicate that the user is an admin, and not tie it to access to any particular admin service.

Reply:
Done